cns/restserver/api_test.go

```diff
@@ -105,15 +105,15 @@ var (
 	errMismatchedNCs = errors.New("GetNetworkContainers failed because NCs not matched")
 
 	nc3 = createOrUpdateNetworkContainerParams{
-		ncID:         "1abc",
+		ncID:         "f47ac10b-58cc-0372-8567-0e02b2c3d477",
 		ncIP:         "10.0.0.5",
 		ncType:       cns.AzureContainerInstance,
 		ncVersion:    "0",
 		podName:      "testpod",
 		podNamespace: "testpodnamespace",
 	}
 	nc4 = createOrUpdateNetworkContainerParams{
-		ncID:         "2abc",
+		ncID:         "f47ac10b-58cc-0372-8567-0e02b2c3d476",
```

Comment:
can we have one testcase with swift prefix?

Reply:
no, the swift prefix is already added during the relevant tests. this represents the NMA NCID which will never be prefixed